docs/support.rst

```diff
@@ -0,0 +1,207 @@
+
+`Validating feature support for USD files <https://developer.apple.com/documentation/realitykit/validating-usd-files?changes=_3>`_
+documents feature support for USD across the different renderers available.
+
+ModelIO
+-------
+
+`ModelIO <https://developer.apple.com/documentation/modelio>`_ is a framework for importing, exporting and manipulating
+common 3D formats.
+
+Autodesk
+========
+
+Autodesk is a developer of several 3D Content Creation, Rendering and CAD software packages.
+
+`Autodesk's USD Website <https://makeanything.autodesk.com/usd>`_ highlights their USD work.
+
+3ds Max
+-------
+
+`3ds Max <https://www.autodesk.com/products/3ds-max/overview>`_ is general purpose 3D package.
+
+`3ds Max USD Documentation <https://knowledge.autodesk.com/support/3ds-max/learn-explore/caas/CloudHelp/cloudhelp/2022/ENU/3dsMax-USD/files/GUID-04F1DF51-0079-4DF8-8457-5AD12B6C0673-html.html>`_
+
+Arnold
+------
+
+`Arnold <https://www.autodesk.com/products/arnold/overview>`_ is a path-traced renderer.
+
+`Arnold USD Documentation <https://docs.arnoldrenderer.com/display/A5ARP/Introduction+to+Arnold+USD>`_
+
+
+Maya
+----
+
+`Maya <https://www.autodesk.com/products/maya/overview>`_ is a general purpose 3D package
+
+Maya USD
+^^^^^^^^
+
+Maya comes bundled with an `open source plugin <https://github.com/Autodesk/maya-usd>`_ .
+
+`Maya USD Documentation <https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2022/ENU/Maya-USD/files/GUID-9E9D45F2-4DA9-497B-8D69-1573ED6B2BA8-html.html>`_
+
+Multiverse for Maya
+^^^^^^^^^^^^^^^^^^^
+
+`Multiverse <https://j-cube.jp/solutions/multiverse/>`_ is another USD plugin for Maya that adds other pipeline features
+to Maya as well.
+
+Blender
+=======
+
+`Blender <https://www.blender.org>`_ is a free and open source 3D creation suite.
+
+`Blender USD Documentation <https://docs.blender.org/manual/en/latest/files/import_export/usd.html>`_
+
+
+Foundry
+=======
+
+The Foundry are a developer of 3D Content Creation, Lighting and Compositing software.
+
+Katana
+------
+
+`Katana <https://www.foundry.com/products/katana>`_ is  look development and lighting package.
+
+Katana has `open source USD plugins <https://learn.foundry.com/katana/dev-guide/Plugins/KatanaUSDPlugins/index.html>`_ as
+well as a support for `Hydra render delegates <https://learn.foundry.com/katana/dev-guide/Plugins/HydraRenderDelegates/index.html>`_.
+
+Modo
+----
+
+`Modo <https://www.foundry.com/products/modo>`_ is a general purpose 3D package.
+
+Nuke
+----
+
+`Nuke <https://www.foundry.com/products/nuke-family/nuke>`_ is a compositing package with support for both USD and Hydra.
+
+`Nuke documentation <https://learn.foundry.com/nuke/content/comp_environment/3d_compositing/usd.html>`_
+
+ZBrush
+------
+
+`ZBrush <https://www.maxon.net/en/zbrush>`_ is a 3D sculpting and painting application.
+
+`ZBrush USD documenation <http://docs.pixologic.com/user-guide/zbrush-other-programs/usd-format/>`_
```

Comment:
This could be under Maxon category, since Zbrish is owned by them now.

Reply:
Ah yeah you're right. I meant for it to be under Maxon but accidentally put it under Foundry